client/src/components/Home.js

```diff
@@ -221,14 +227,52 @@ function Home(props) {
               .then(response => response.json())
               .then(data => {
                   console.log(data)
+                  eventId = data._id;
               })
-          // Empty the input fields
-          setNewEvent({});
-          // Close the Modal
-          setNewEventModal(false);
-          // Update events list by toggling the boolean
-          toggleUpdateEvents();
-          toasts.showToastSuccessMessage("Event created successfully!");
+            if(!selectedFile){
+              // Empty the input fields
+              setNewEvent({});
+              // Close the Modal
+              setNewEventModal(false);
+              // Update events list by toggling the boolean
+              toggleUpdateEvents();
+              toasts.showToastSuccessMessage("Event created successfully!");
+              return;
+            }  
+            if (selectedFile.type !== 'image/png' && selectedFile.type !== 'image/jpeg') {
+              // Empty the input fields
+              setNewEvent({});
+              // Close the Modal
+              setNewEventModal(false);
+              // Update events list by toggling the boolean
+              toggleUpdateEvents();
+              toasts.showToastMessage('The event is created without an image');
+              return;
+            }
+            const formData = new FormData();
+            formData.append('image', selectedFile);
+            const response = await fetch(`/api/updateImage/${eventId}`, {
+              method: 'POST',
+              body: formData
+            });
+            if(response.status == 413){
+              toasts.showToastMessage("the event is created but the image is to big");
```

Comment:
same here

Reply:
The first letter of a sentence should also be capitalized and all of these cases

Reply:
Also 'was created'